### ISSUE TITLE
Make `--output-prefix` optional (or remove it entirely)

### ISSUE THREAD

**Issue by randomPoison** [immunant/IA2-Phase2#482]
Currently the `--output-prefix` flag on the rewriter is required to be specified. I'm not sure why we need that prefix to be specified by the user, it seems like we could use a default prefix of `ia2_` or `wrapper_` (or something similar). If there's a reason why we need to give the user an option to control that prefix, then I think we should make `--output-prefix` optional, though I think the better solution would be to remove it entirely if that wouldn't cause any problems.

**Comment by kkysen** [comment]
I agree.  Having this option doesn't seem very useful, and it also makes referring to these files in documentation, etc. more confusing.  It should also go in `--output-directory` (I forgot if it already does, since I just pass an absolute path).

**Comment by randomPoison** [comment]
It currently puts those files in whatever directory you run the rewriter in. I agree they should go in the output directory, but when I brought that up the other day I think @ayrtonm said that doesn't make sense for some reason (I don't remember what exactly).
